pyproject.toml

```diff
@@ -45,6 +45,7 @@ dev = [
 io = ["netCDF4", "h5netcdf", "scipy", 'pydap; python_version<"3.10"', "zarr", "fsspec", "cftime", "pooch"]
 parallel = ["dask[complete]"]
 viz = ["matplotlib", "seaborn", "nc-time-axis"]
+extension-arrays = ["plum-dispatch"]
```

Comment:
https://github.com/pydata/xarray/pull/8723#issuecomment-1937057834 @TomNicholas done already!  There's even a check in the code - no `plum` and you get the numpy fallback of all this new functionality i.e., exactly what happened before.